vm/src/stdlib/os.rs

```diff
@@ -2436,6 +2436,53 @@ mod posix {
         Ok(())
     }
 
+    #[pyattr]
+    #[pyclass(name = "sched_param")]
+    #[derive(Debug)]
+    struct SchedParam {
+        sched_priority: PyObjectRef,
+    }
+
+    impl PyValue for SchedParam {
+        fn class(_vm: &VirtualMachine) -> &PyTypeRef {
+            Self::static_type()
+        }
+    }
+
+    impl TryFromObject for SchedParam {
+        fn try_from_object(_vm: &VirtualMachine, obj: PyObjectRef) -> PyResult<Self> {
+            Ok(SchedParam {
+                sched_priority: obj,
+            })
+        }
+    }
+
+    #[pyimpl]
+    impl SchedParam {
+        #[pyslot]
+        fn tp_new(cls: PyTypeRef, args: FuncArgs, vm: &VirtualMachine) -> PyResult {
+            let obj: PyObjectRef = args.bind(vm)?;
```

Comment:
Recently `SlotConstructor` is added. I recommend to use it for new code rather than `bind`

Reply:
Because you already added `TryFromObject for SchedParam`,
```rust
        fn tp_new(cls: PyTypeRef, args: FuncArgs, vm: &VirtualMachine) -> PyResult {
            let obj: SchedParam = args.bind(vm)?;
```
will work.